app/templates/live.hbs

```diff
@@ -8,59 +8,52 @@
   />
   {{! TODO - add more else if statement instead of only else }}
   {{#if (eq this.activeTab 'Screenshare')}}
-    {{#if this.liveService.isJoined}}
-      <div class='live__container'>
-        <div class='live__video-container' data-test-video-container>
-          {{! TODO - add skeleton to the whole structure }}
-          {{#if this.isLoading}}
-            <Reusables::Skeleton />
-          {{else}}
-            {{! TODO - Remove this dummy src and it's video }}
-            <VideoScreen @src='/assets/birds_footage.mp4' />
```

Comment:
Can we please remove this resource from the assets ?
Also there is one host.svg that is also not utilized I guess, can we please remove that as well ?